lib/matplotlib/backends/backend_wx.py

```diff
@@ -342,38 +341,19 @@ def get_gc(self):
         return self.gc
 
     def get_wx_font(self, s, prop):
-        """
-        Return a wx font.  Cache instances in a font dictionary for
-        efficiency
-        """
+        """Return a wx font.  Cache font instances for efficiency."""
         DEBUG_MSG("get_wx_font()", 1, self)
-
         key = hash(prop)
-        fontprop = prop
-        fontname = fontprop.get_name()
-
         font = self.fontd.get(key)
         if font is not None:
             return font
-
-        # Allow use of platform independent and dependent font names
-        wxFontname = self.fontnames.get(fontname, wx.ROMAN)
-        wxFacename = ''  # Empty => wxPython chooses based on wx_fontname
-
         # Font colour is determined by the active wx.Pen
         # TODO: It may be wise to cache font information
-        size = self.points_to_pixels(fontprop.get_size_in_points())
-
-        font = wx.Font(int(size + 0.5),             # Size
-                       wxFontname,                # 'Generic' name
-                       self.fontangles[fontprop.get_style()],   # Angle
-                       self.fontweights[fontprop.get_weight()],  # Weight
-                       False,                     # Underline
-                       wxFacename)                # Platform font name
-
-        # cache the font and gc and return it
-        self.fontd[key] = font
-
+        self.fontd[key] = font = wx.Font(  # Cache the font and gc.
+            pointSize=round(self.points_to_pixels(prop.get_size_in_points())),
```

Comment:
This doesn't round the same, but I'm not sure if it's important:
```python
In [9]: for i in range(10): 
   ...:     i = i + 0.5 
   ...:     print(i, round(i), int(i + 0.5)) 
   ...:                                                                                                                                               
0.5 0 1
1.5 2 2
2.5 2 3
3.5 4 4
4.5 4 5
5.5 6 6
6.5 6 7
7.5 8 8
8.5 8 9
9.5 10 10
```

Reply:
Actually, upon further testing wx.Font accepts floats just fine here so I just dropped the rounding (after all freetype accepts sizes in multiples of (1/64) so there's no reason to drop that info...).